Sprint-2/debug/address.js

```diff
@@ -12,4 +12,5 @@ const address = {
   postcode: "XYZ 123",
 };
 
-console.log(`My house number is ${address[0]}`);
+console.log(`My house number is ${address.houseNumber}`);
```

Comment:
Great! just want to mention that you can access object with brackets as well like this: `address['houseNumber']`.